scripter/lua/lua.go

```diff
@@ -18,6 +19,8 @@ var (
 	_ = scripter.Register("lua", New)
 )
 
+const cleanupTimer = 60
```

Comment:
`const` variables are usually all uppercase (`CLEANUP_TIMER`) to indicate that they are a constant.